### PR TITLE
OP-780 removing delete column enabling active

### PR DESCRIPTION
**Pull request by AndreiDodu**
enabling active column for all entities

### CODE REVIEW

File: src/main/java/org/isf/medstockmovtype/model/MovementType.java [hunk 1]
```diff
@@ -30,16 +30,15 @@
...
 import org.springframework.data.jpa.domain.support.AuditingEntityListener;
 
 /**
- * ------------------------------------------
- * MovementType - model for the movement type
- * -----------------------------------------
- * modification history
- * ? - bob - first version
- * 18/01/2015 - Antonio - ported to JPA
+ * ------------------------------------------ MovementType - model for the
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medstockmovtype/model/MovementType.java:39]: I wasn't going to comment about all the space/reformatting going on but here again a nicely formatted comment block is mashed together making it hard to read.

File: src/main/java/org/isf/patient/model/Patient.java [hunk 2]
```diff
@@ -76,133 +78,126 @@
...
-	 * (cognome) PAT_AGE int NOT NULL , --age PAT_SEX char (1) NOT NULL , --sex :
-	 * M or F PAT_ADDR varchar (50) NULL , --address (via , n.) PAT_CITY varchar
-	 * (50) NOT NULL , --city PAT_NEXT_KIN varchar (50) NULL , --next kin
-	 * (parente prossimo, figlio di..) PAT_TELE varchar (50) NULL , --telephone
-	 * number PAT_MOTH char (1) NULL , --mother: D=dead, A=alive PAT_FATH char
-	 * (1) NULL , --father: D=dead, A=alive PAT_LEDU char (1) NULL , --level of
-	 * education: 1 or 2 or 3 or 4 PAT_ESTA char (1) NULL , --economic status:
-	 * R=rich, P=poor PAT_PTOGE char (1) NULL , --parents together: Y or N
-	 * PAT_LOCK int NOT NULL default 0, PRIMARY KEY ( PAT_ID )
+	 * --first name (nome) PAT_SNAME varchar (50) NOT NULL , --second name (cognome)
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/patient/model/Patient.java:86]: again formatting

File: src/main/java/org/isf/patient/service/PatientIoOperations.java [hunk 3]
```diff
@@ -36,19 +36,14 @@
...
- * 05/05/2005 - giacomo  - first beta version
- * 03/11/2006 - ross - added toString method. Gestione apici per
- * nome, cognome, citta', indirizzo e note
- * 11/08/2008 - alessandro - added father & mother's names
- * 26/08/2008 - claudio    - added birth date
- * modified age
- * 01/01/2009 - Fabrizio   - changed the calls to PAT_AGE fields to
- * return again an int type
- * 03/12/2009 - Alex       - added method for merge two patients history
+ * ------------------------------------------ PatientIoOperations - dB
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/patient/service/PatientIoOperations.java:39]: and again ....

File: src/main/java/org/isf/utils/db/Auditable.java [hunk 4]
```diff
@@ -56,8 +57,9 @@
     @Column(name="LAST_MODIFIED_DATE")
     protected LocalDateTime lastModifiedDate;
    
+	@NotNull
     @Column(name="ACTIVE")
-    protected Integer active = 1;
+    protected Integer active = Integer.valueOf(1);
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/utils/db/Auditable.java:62]: The code is correct as it was; you've added unnecessary boxing,  Explicit manual boxing is unnecessary starting with Java 5 and can safely be removed.  Please see if there are other occurrences.  Thanks.

**Comment by mwithi** [comment]
Hi @AndreiDodu did you check why 3 tests are failing? https://github.com/informatici/openhospital-core/runs/6539349938?check_suite_focus=true

**Comment by AndreiDodu** [comment]
> Hi @AndreiDodu did you check why 3 tests are failing? https://github.com/informatici/openhospital-core/runs/6539349938?check_suite_focus=true

I fixed some tests but I need some functional information for the testIoNewAutomaticDischargingMovementDifferentLots.

**Pull request closed without merging**